### ISSUE TITLE
Has anyone even run the Python code? So many errors...

### ISSUE THREAD

**Issue by TimothyBramlett** [stripe-samples/subscription-use-cases#19]
I am not even going to list them all here.  Just try to run server.py

setup_intent is not defined.

And nowhere do you list the other environmental variables the script depends on.

**Comment by ctrudeau-stripe** [comment]
Hello @TimothyBramlett, there are step by step how to get the sample running in the [main readme](https://github.com/stripe-samples/subscription-use-cases#how-to-run-locally). Once you copy and the .env.example into the server/python folder. There are python specific instructions [here](https://github.com/stripe-samples/subscription-use-cases/tree/master/fixed-price-subscriptions/server/python#how-to-run). You'll need to get your Stripe publishable and secret key.

**Comment by TimothyBramlett** [comment]
@ctrudeau-stripe Did you not even read what I wrote?  I cannot believe how bad Stripes documentation and sample code has gotten in the years since I first wrote code against your apis.  Look at line 53 of server.py:

![image](https://user-images.githubusercontent.com/7890441/86036399-675c6c80-ba03-11ea-8de5-2e7508d95838.png)

Nowhere is setup_intent defined.  I had to look at usage-based-subscriptions/server/python/server.py to find a hint into what needs to happen.

And your documentation in general is littered with missing links like this page here: https://stripe.com/docs/billing/subscriptions/fixed-price#what-youll-build

Plenty of 404s when you click on view_sample_code. And the examples on that page do not even match what is in the repo now.

For a developer focused product like Stripe the documentation and sample code are INCREDIBLY important.  Your documentation and sample code need to match exactly.  They actually do more harm than good right now.

**Comment by ctrudeau-stripe** [comment]
Nice catch! :slightly_smiling_face: The setupIntent was there from when we were having trials in the guide you linked but we removed them recently and we didn’t remove that line from the sample. We added a react frontend to the examples leading to some 404s, this will be deployed shortly. We are still iterating on this latest guide and sample. Thank you for the feedback! Also, if you’d like to chat more with us, we’re available on IRC: irc://irc.freenode.net/stripe.
